chunks/modelfreemodels.ipynb

```diff
@@ -229,11 +229,11 @@
     "\n",
     "When might we use these?\n",
     "\n",
-    "* If the data being modelled really are suspected to consist of multiple populations\n",
+    "* If the data being modeled really are suspected to have come from multiple origins\n",
     "\n",
     "e.g.\n",
-    "- supernova luminosities (without spectroscopic typing)\n",
-    "- source vs. background photon energies"
+    "- supernova luminosities $L$ (without spectroscopic typing $T$), conditional (prior) PDF $P(L|T)$\n",
+    "- source vs. background photon energies $E$, sampling distribution $P(N|T)$, $P(E|T)$"
```

Comment:
Ah yes - good. Will fix this. I was trying to make the mixture model a little more concrete, to show *which* PDFs it makes sense to model with a mixture. The examples seem to show neatly that either the sampling distribution or the conditional PDF are fair game for thinking about in terms of mixtures.